### DISCUSSION TITLE
Custom aspect-ratio breakpoint messes up text sizes

### DISCUSSION THREAD

**Discussion by atsakiridis** [tailwindlabs/tailwindcss#13618] [Help]
Hello,

I introduced the following breakpoints in tailwind meant to respond to aspect-ratio changes:

```
  theme: {
    extend: {
      screens: {
        'myns-mid': { 'raw': '(min-aspect-ratio: 2/3)' },
        'myns-wide': { 'raw': '(min-aspect-ratio: 1/1)' },
      },
```

Once I make this config change, and before I even use it in my react components `className` the text size of some irrelevant element gets smaller for some reason

Notice that:
- Otherwise the config change works fine once I start using it
- I cannot reproduce in play.tailwindcss.com, unfortunately
- Removing the TW config and adding it as normal css is working without side-effects

Thanks in advance

**Comment by wongjn** [comment]
Please consider providing a project that reproduces the unexpected behavior.
